src/univalue/include/univalue.h

```diff
@@ -1,30 +1,27 @@
 // Copyright 2014 BitPay Inc.
 // Copyright 2015 Bitcoin Core Developers
 // Distributed under the MIT software license, see the accompanying
-// file COPYING or https://www.opensource.org/licenses/mit-license.php .
+// file COPYING or https://opensource.org/licenses/mit-license.php.
 
-#ifndef UNIVALUE_H__
-#define UNIVALUE_H__
+#ifndef __UNIVALUE_H__
+#define __UNIVALUE_H__
```

Comment:
These are technically reserved names but I would like to keep them the same as upstream so we don't have to apply any patches when we update it in the future.

Reply:
Strictly speaking [this is undefined behaviour](https://stackoverflow.com/a/228797/393146). What's our general policy about undefined behaviour in dependencies? We avoid reserved names elsewhere in the source: 84738627ce32767bacb86e1ba109147fab29cafb, #1272, #2767.

Reply:
Since it only affects these two lines, I think the maintenance burden of differing here is negligible. I filed a PR against upstream Univalue: https://github.com/jgarzik/univalue/pull/71